client/src/utilities/keycloak.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-underscore-dangle */
+import Raven from 'raven-js';
+import queryString from 'querystringify';
+
+export function token(keycloak) {
+  return new Promise(resolve =>
+    keycloak
+      .updateToken()
+      .success(() => resolve(keycloak.token))
+      .error(() => {
+        // Redirect to login page
+        keycloak.login();
+      })
+  );
+}
+
+export function refreshToken(keycloak) {
+  if (keycloak == null) {
+    throw new Error('Keycloak not initialized');
+  }
+  return keycloak.refreshToken;
+}
+export function login(keycloak) {
+  if (keycloak == null) {
+    throw new Error('Keycloak not initialized');
+  }
+  return keycloak.login();
+}
+// eslint-disable-next-line no-unused-vars
+export function logout(keycloak) {
+  throw new Error('Keycloak dont have this logout functionality');
+}
+
+export function init(env, keycloak) {
+  const { tenant, sentryDSN } = env;
+  return new Promise((resolve, reject) => {
+    if (process.env.NODE_ENV === 'production') {
+      Raven.config(sentryDSN).install();
```

Comment:
Dont we want to do also this in the case of Auth0?

Reply:
solved by https://github.com/akvo/akvo-lumen/pull/2244/commits/d5542dae5efc10b3a3a9fd6ba11d1ad92350bd9b